File: Data Structures & Algorithms/sliding-window-maximum/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} k
     * @return {number[]}
     */
    maxSlidingWindow(nums, k) {
        const arr = [];

        let l = 0;
        let r = k - 1;

        while (r < nums.length) {
            let maxNum = nums[l];

            for (let i = l; i <= r; i++) {
                maxNum = Math.max(maxNum, nums[i]);
            }

            arr.push(maxNum);

            l++;
            r++;
        }

        return arr;

    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-1.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        maxC = 0
        count = 0
        for num in nums:
            if num == 1:
                count += 1
                maxC = max(maxC,count)
            else:
                count = 0
        return maxC
